pages/item/_itemId/index.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-container fluid class="pa-0 itemContainer">
+    <v-img
+      :src="getItemBackdrop(item.Id)"
```

Comment:
Is there a way to make this responsive?

Currently if I resize the window, it stays with the same image. I'd like to avoid non-responsive stuff as much as possible, when possible.

Reply:
We can use this, probably: https://vuetifyjs.com/en/directives/resizing/

Reply:
Oh yeah I was looking for a function like that, but never managed to find one. I'll implement it now.